src/OpenDB/test/unitTestsPython/TestITerm.py

```diff
@@ -51,32 +51,32 @@ def test_connection_from_inst(self):
         self.assertEqual(n.getITerms(), [])
         self.assertIsNone(self.iterm_a.getNet())
         self.assertFalse(self.iterm_a.isConnected())
-    def test_avgxy_R0(self):
-        x = odb.new_int(0)
-        y = odb.new_int(0)
-        self.assertFalse(self.iterm_a.getAvgXY(x, y))   #no mpin to work on
-        mterm_a = self.and2.findMTerm('a')
-        mpin_a = odb.dbMPin_create(mterm_a)
-        self.assertFalse(self.iterm_a.getAvgXY(x, y))   #no boxes to work on
-        geo_box_a_1 = odb.dbBox_create(mpin_a, self.lib.getTech().getLayers()[0], 0, 0, 50, 50)
-        self.assertTrue(self.iterm_a.getAvgXY(x, y))
-        self.assertEqual(odb.get_int(x), int((0+50)/2))
-        self.assertEqual(odb.get_int(y), int((0+50)/2))
-        geo_box_a_2 = odb.dbBox_create(mpin_a, self.lib.getTech().getLayers()[0], 5, 10, 100, 100)
-        self.assertTrue(self.iterm_a.getAvgXY(x, y))
-        self.assertEqual(odb.get_int(x), int( ((0+50)+(5+100))/4 ) )
-        self.assertEqual(odb.get_int(y), int( ((0+50)+(10+100))/4 ) )
-    def test_avgxy_R90(self):
-        x = odb.new_int(0)
-        y = odb.new_int(0)
-        mterm_a = self.and2.findMTerm('a')
-        mpin_a = odb.dbMPin_create(mterm_a)
-        geo_box_a_1 = odb.dbBox_create(mpin_a, self.lib.getTech().getLayers()[0], 0, 0, 50, 50)
-        geo_box_a_2 = odb.dbBox_create(mpin_a, self.lib.getTech().getLayers()[0], 0, 0, 100, 100)
-        self.inst.setOrient('R90')
-        self.assertTrue(self.iterm_a.getAvgXY(x, y))
-        self.assertEqual(odb.get_int(x), int( ((0+50)+(0+100))/4 )*-1 )
-        self.assertEqual(odb.get_int(y), int( ((0+50)+(0+100))/4 ) )
+    # def test_avgxy_R0(self):
+    #     x = odb.new_int(0)
```

Comment:
Why is this section commented out?

Reply:
odb.new_int is removed from the swig helper functions and used here. OpenLANE are suffering from similar problem and I am working on the fix on a separate thread

Reply:
#1028